File: vmc/beginning/l 2.6/compute_change.c
#include<stdio.h>
int compute_change(int x)
{
    if (x<100)
    {
        int y = 100-x;
        int q = y/25;
        printf("%d,%s\n",q , "Quarter");
        int d = (y-q*25)/10;
        printf("%d,%s\n",d , "dime");
        int n = (y-q*25-d*10)/5;
        printf("%d,%s\n",n , "nickel");
        int p = (y-q*25-d*10-n*5);
        printf("%d,%s\n",p , "Penny");

    }
}
int main()
{
    int x;
    printf("Enter your x :");
    scanf("%d",&x);
    compute_change(x);
}

File: vm/beginning/l 2.6/ComputeChange.py
def compute_change(x):
    if x < 100 :
        y = 100-x
        q = y//25
        print(q , "Quarter")
        d = (y-q*25)//10
        print(d , "dime")
        n = (y-q*25-d*10)//5
        print(n , "nickel")
        p = (y-q*25-d*10-n*5)
        print(p , "Penny")



while True:
    x = int(input("pleas inter number:"))
    compute_change(x)